trivy-task/task.json

```diff
@@ -63,6 +63,26 @@
       "groupName": "trivyRunner",
       "visibleRule": "docker = true"
     },
+    {
+    "name": "overrideDockerImage",
```

Comment:
@owenrumney , 
Hi! 
Is there any reason to add 2 inputs for this feature?
`dockerImage` input with default value `aquaseq/trivy`  should be enough.

Reply:
and probably it's better to use `trivyImage` or `trivyDockerImage` instead of `dockerImage` to differentiate from `image`

Reply:
Hey @bonddim  - yeah I've gone with one, good point about switching to `trivyImage`